### PR TITLE
Gallery upgrades

### PR DESCRIPTION
**Pull request by jrleeman**
* Tries to hide the thumbnail selection down in the example code.
* flake updates to the MM#7
* Add MetPy Mondays to flake8 checks.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,6 +35,7 @@ install:
 
 script:
   - flake8 examples
+  - flake8 metpy_mondays
```
Comment (dopplershift) [inline .travis.yml]: Can just do: `flake8 examples metpy_mondays`

Comment (jrleeman) [inline .travis.yml]: Will do.

**Comment by jrleeman** [comment]
Ready to merge if it looks good to you now @dopplershift ?

**Comment by dopplershift** [comment]
LGTM

**Pull request merged**